docs/snippets/vue/portable-stories-vitest-override-globals.ts.mdx

```diff
@@ -0,0 +1,24 @@
+```ts
+// Button.test.ts
+import { test } from 'vitest';
+import { render } from '@testing-library/vue';
+import { composeStory } from '@storybook/vue3';
+
+import meta, { Primary } from './Button.stories';
+
+test('renders in English', async () => {
+  const PrimaryStory = composeStory(
+    Primary,
+    meta,
+    { globalTypes: { locale: 'en' } } // 👈 Project annotations to override the locale
```

Comment:
Turns out there was a bug in portable stories which didn't allow users to use `globals` and only use `globalTypes`, but that doesn't really make sense. Now that is fixed, so we need to update to:

```ts
{ globals: { locale: 'en' } }
```

This matches the API of which users are familiar with in a story render function:
```ts
export const MyStory = (args, { globals }) => {}
```

instead of the API for defining toolbars in .storybook/preview (which will also work, but does not feel right):
```
export default {
  globalTypes: {
    locale: {  defaultValue: 'en' }
  }
}
```